### PR TITLE
fake flood fix

### LINKED ISSUE #49806: fake plasma flood from RDS(reality disassociation syndrome) blocks clicking on things
Issue reported from  Round ID: 131738 (/tg/Station Manuel [ENGLISH] [US-WEST] [ROLEPLAY] [99% LAG FREE])

Reporting client version: 513.1507

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request

fake plasma flood hallucination no more blocks clicking on things
fix #49806

## Why It's Good For The Game

Bugs bad.

## Changelog
:cl:
fix: fake plasma flood hallucination no more blocks clicking on things
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
check if you can see them in the right click list

**Comment by Dennok** [comment]
> check if you can see them in the right click list

Cant see it in right click list

**Comment by XDTM** [comment]
great, should be fine then

File: code/modules/flufftext/Hallucination.dm [hunk 1]
```diff
@@ -167,10 +175,12 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 		qdel(src)
 		return
 	feedback_details += "Vent Coords: [center.x],[center.y],[center.z]"
-	var/image/plasma_image = image(image_icon,center,image_state,FLY_LAYER)
+	var/obj/effect/plasma_image_holder/PIH = new(center)
```
Comment (ShizCalev) [inline code/modules/flufftext/Hallucination.dm]: lowercase please

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/flufftext/Hallucination.dm (hunk 1 above)
Comment (XDTM) [reply code/modules/flufftext/Hallucination.dm]: don't we allow uppercase acronyms for temp variables?

**Pull request merged**